Frameworks/UIKit/UIView.mm

```diff
@@ -2252,14 +2251,18 @@ - (CGPoint)convertPoint:(CGPoint)pos fromView:(UIView*)fromView {
 */
 - (void)drawLayer:(CALayer*)layer inContext:(CGContextRef)context {
     UIGraphicsPushContext(context);
+    _CGContextPushBeginDraw(context);
+
+    auto popEnd = wil::ScopeExit([context]() {
+        _CGContextPopEndDraw(context);
+        UIGraphicsPopContext();
+    });
 
     CGRect bounds;
     // TODO(DH)
     bounds = self.bounds;
-    //bounds = CGContextGetClipBoundingBox(context);
+    // bounds = CGContextGetClipBoundingBox(context);
```

Comment:
nit: remove commented out line.  Also, the whole thing could be CGRect bounds = self.bounds, or even just [self drawRect:self.bounds] since it is not being reused.